client/src/components/snippets/edit/ClaudeFileEditor.tsx

```
@@ -0,0 +1,417 @@
+          {/* Template Grid */}
+          <div className="flex-1 overflow-y-auto p-6">
+            <p className="text-sm text-light-text dark:text-dark-text mb-6 opacity-75">
+              Select a Claude Code file type to start with a template, or skip to create a blank file.
+            </p>
+            <div className="grid grid-cols-2 gap-3">
+              {templates.map((template) => (
+                <button
+                  key={template.value}
+                  onClick={() => handleTemplateSelect(template.value)}
+                  className="flex items-center gap-3 p-4 rounded-lg border-2 border-light-border dark:border-dark-border hover:border-light-primary dark:hover:border-dark-primary hover:bg-light-hover dark:hover:bg-dark-hover transition-all text-left"
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)